### PR TITLE
Further updates after the Cursed possessions update

### PR DESCRIPTION
**Pull request by BotiKis**
## Includes
- New learned secondary evidences for Hantu, Banshee, The Mimic, Mare and The Twins.
- Clarifications + details to existing secondary evidences for all ghosts
- New Ouija board questions from 0.4 and 0.5

### Sources
- [Psychos research](https://www.youtube.com/c/Psychoooo/videos)
- [Insyms play and dev talk](https://www.youtube.com/channel/UC5uNya42ayhsRnZOR3mO6NA)

### CODE REVIEW

File: src/phasmophobia-data/data.js [hunk 1]
```diff
@@ -1,5 +1,83 @@
...
+		"Head mounted cameras, motion sensors and sound sensors do not count towards the speed boost",
+		"Can initiate a hunt at an avarage sanity of 70% if there is electrical equipment nearby",
+		"Reduced chance to leave fingerprints",
+		"Low chance to leave fingerprints with 6 fingers",
+		"Fingerprints have a 50% chance to disappear earlier (60s) than of other ghosts (120s)",
+		"Always gives the Fingerprints evidence on nightmare difficulty",
+		"Has ghost orbs as 4th evidence",
+		"Picks a ghost to mimic and changes it occasionally",
+		"Copies all abilities and behaviour of the picked ghost"
+	],
```
Comment (un0btanium) [inline src/phasmophobia-data/data.js:80]: You dont require those anymore. In order to reduce redundancy, I removed these two arrays from the data.js file in a recent commit. They are now automatically generate at website startup from the ghosts primary and secondary evidences instead.

File: src/phasmophobia-data/data.js [hunk 2]
```diff
@@ -53,45 +131,53 @@ var data = {
...
 			"primaryEvidences": [
 				"Freezing Temperature",
 				"EMF Level 5",
-				"Fingerprints",
-			],
-			"secondaryEvidences": [
-				"During a hunt it travels faster when chasing a target until it comes close",
+				"Fingerprints"
+      ],
+      "secondaryEvidences": [
```
Comment (un0btanium) [inline src/phasmophobia-data/data.js:151]: Intendation is messed up a little.

File: src/phasmophobia-data/data.js [hunk 3]
```diff
@@ -242,12 +334,12 @@ var data = {
...
 				"EMF Level 5",
 				"Fingerprints",
-				"Ghost Orbs",
+				"Ghost Orbs"
 			],
 			"secondaryEvidences": [
 				"Reduced chance to leave fingerprints",
 				"Low chance to leave fingerprints with 6 fingers",
-		    	"Fingerprints disappear earlier (60s) than other ghosts (120s)",
+        "Fingerprints have a 50% chance to disappear earlier (60s) than of other ghosts (120s)",
```
Comment (un0btanium) [inline src/phasmophobia-data/data.js:342]: Intendation

File: src/phasmophobia-data/data.js (hunk 1 above)
Comment (BotiKis) [reply src/phasmophobia-data/data.js:80]: Wow good to know!

**Comment by BotiKis** [comment]
I completely missed, that I had to resovle stuff here, apologies!

**Comment by BotiKis** [comment]
> Hey, no problem. Did you have time to make the changes yet? edit: I dont think resolving changes the code automatically ☺

Ooooh well... silly me!
On it!

File: src/phasmophobia-data/data.js (hunk 2 above)
Comment (BotiKis) [reply src/phasmophobia-data/data.js:151]: Don't know what is going on here. They are normal tabs and look well in atom. maybe just a github visual glitch?

Screenshot 2022-01-05 at 19 09 04

File: src/phasmophobia-data/data.js (hunk 3 above)
Comment (BotiKis) [reply src/phasmophobia-data/data.js:342]: Same as above @un0btanium 

Screenshot 2022-01-05 at 19 11 41

File: src/phasmophobia-data/data.js (hunk 2 above)
Comment (un0btanium) [reply src/phasmophobia-data/data.js:74]: Hm, strange indeed. I will check on my end as well and fix if there is anything to fix.

**Comment by un0btanium** [comment]
@BotiKis spooky intendations. Any ghosts here? 😂

**Comment by BotiKis** [comment]
@un0btanium where are you, how old are you, give us a `tab`. 😏

**Pull request merged**